src/link.js

```diff
@@ -199,13 +198,81 @@ export default class Link extends Feature {
 			callback: () => this._hidePanel()
 		} );
 
-		// Append panel element to body.
 		editor.ui.add( 'body', balloonPanel );
 
 		return balloonPanel;
 	}
 
 	/**
+	 * Creates the {@link link.ui.LinkForm} instance.
+	 *
+	 * @private
+	 * @returns {link.ui.LinkForm} Link form instance.
+	 */
+	_createForm() {
```

Comment:
I think that we could move this to the `LinkForm` class (to the controller). This will allow to reuse that whole component (e.g. in the "link image" feature).